Backend/app/dashboard/provider-dashboard/src/API/ProviderPlatform/DynamicOfferDriver/Ride.hs

```diff
@@ -71,12 +81,25 @@ type RideSyncAPI =
   ApiAuth 'DRIVER_OFFER_BPP 'RIDES 'RIDE_SYNC
     :> Common.RideSyncAPI
 
+buildTransactionMultipleRide ::
+  ( MonadFlow m,
+    Common.HideSecrets request
+  ) =>
+  Common.RideEndpoint ->
+  ApiTokenInfo ->
+  Maybe request ->
+  m DT.Transaction
+buildTransactionMultipleRide endpoint apiTokenInfo =
+  T.buildTransaction (DT.RideAPI endpoint) (Just DRIVER_OFFER_BPP) (Just apiTokenInfo) Nothing Nothing
```

Comment:
Do not create separate function for this. Instead simply modify existing one, so it will apply to all cases:

```
buildTransaction ::
  ( MonadFlow m,
    Common.HideSecrets request
  ) =>
  Common.RideEndpoint ->
  ApiTokenInfo ->
  Maybe (Id Common.Ride) ->
  Maybe request ->
  m DT.Transaction
buildTransaction endpoint apiTokenInfo rideId =
  T.buildTransaction (DT.RideAPI endpoint) (Just DRIVER_OFFER_BPP) (Just apiTokenInfo) Nothing rideId
```